### ISSUE TITLE
📖 Update docs/README.md with virtualcluster flavor

### ISSUE THREAD

**Issue by christopherhein** [kubernetes-sigs/cluster-api-provider-nested#141]
**User Story**

As an operator I would like to use `clusterctl` to deploy a VC backed CAPN by following the quickstart so I don't need to have tribal knowledge.

**Detailed Description**

After #136 is merged and a release is cut we should make sure we update the readme to reflect deploying CAPN + VC as an option.

/kind documentation
/milestone v0.1.x

**Comment by christopherhein** [comment]
hey @dongwangdw we probably need to make sure we have #129 and #130 done for this to be done properly just an FYI as this might be on hold.

**Comment by dongwangdw** [comment]
@christopherhein 
OK. Thanks for the info. It seems there are couples of items should be achived before this one. 
I will have my eye on this.

**Comment by christopherhein** [comment]
Update to use released binary - `curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.0/clusterctl-linux-amd64 -o clusterctl`
